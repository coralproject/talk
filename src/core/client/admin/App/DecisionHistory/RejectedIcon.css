.root {
  color: var(--v2-palette-error-main);
}
