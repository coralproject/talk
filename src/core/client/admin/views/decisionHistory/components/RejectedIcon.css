.root {
  color: var(--palette-error-main);
}
