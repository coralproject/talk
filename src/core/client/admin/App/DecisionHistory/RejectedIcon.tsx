import React, { FunctionComponent } from "react";

import { Icon } from "coral-ui/components";

import styles from "./RejectedIcon.css";

const RejectedIcon: FunctionComponent = () => (
  <Icon size="md" className={styles.root}>
    cancel
  </Icon>
);

export default RejectedIcon;
