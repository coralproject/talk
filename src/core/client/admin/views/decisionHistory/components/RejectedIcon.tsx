import React, { FunctionComponent } from "react";

import { Icon } from "talk-ui/components";

import styles from "./RejectedIcon.css";

const RejectedIcon: FunctionComponent = () => (
  <Icon className={styles.root}>cancel</Icon>
);

export default RejectedIcon;
