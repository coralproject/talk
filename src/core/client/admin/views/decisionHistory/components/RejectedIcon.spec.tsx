import { shallow } from "enzyme";
import React from "react";

import RejectedIcon from "./RejectedIcon";

it("renders correctly", () => {
  const wrapper = shallow(<RejectedIcon />);
  expect(wrapper).toMatchSnapshot();
});
