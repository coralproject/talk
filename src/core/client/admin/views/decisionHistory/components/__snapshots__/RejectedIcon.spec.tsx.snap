// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<withPropsOnChange(Icon)
  className="RejectedIcon-root"
>
  cancel
</withPropsOnChange(Icon)>
`;
